test/NoEnvironmentTests.jl

```diff
@@ -0,0 +1,78 @@
+module NoEnvironmentTests
```

Comment:
what does `NoEnvironmentTests` mean? 

the code ressembles very much the one of `OctreeDistributedDiscreteModelsTests.jl`

Can we perhaps define a common source file that both tests include?